perf.groovy

```diff
@@ -562,20 +560,29 @@ parallel(
 
                     batchFile("tests\\runtest.cmd ${configuration} ${architecture} GenerateLayoutOnly")
 
+                    def runXUnitPerfCommonArgs = "-arch ${arch} -configuration ${configuration} -generateBenchviewData \"%WORKSPACE%\\Microsoft.Benchview.JSONFormat\\tools\" ${uploadString} -runtype ${runType} -outputdir \"${outputDirectory}\" -scenarioTest"
+                    def failedOutputLogFilename = "run-xunit-perf-scenario.log"
+
+                    // Using a centinel file to
+                    batchFile("if exist \"${failedOutputLogFilename}\" del /q /f \"${failedOutputLogFilename}\"")
+                    batchFile("if exist \"${failedOutputLogFilename}\" (echo [ERROR] Failed to previously created \"${failedOutputLogFilename}\" file.& exit /b 1)")
+
```

Comment:
Couple of typos: `centinel` --> `sentinel` and `failed to **delete** previously...`